app/views/admin/users/index.html.haml

```diff
@@ -23,10 +23,8 @@
             %td
               = user.id
             %td
-              - if user.confirmed?
-                confirmed
-              - else
-                unconfirmed
+              - if can? :update, user
+                = check_box_tag user.id, user.id, user.confirmed?, method: :patch, url: "/admin/users/#{user.id}/toggle_confirmation?user[to_confirm]=", class: 'switch-checkbox', data: { size: 'small', off_color: 'warning', on_text: 'Yes', off_text: 'No' }
```

Comment:
This is one very very long line :)
It would be more readable if you could split it in multiple lines.